### PR TITLE
using prelude rather than ecs::xyz in derive

### PR DESCRIPTION
**Pull request by ekalosak**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,11 +15,11 @@ homepage = "https://github.com/zkat/big-brain"
 [workspace]
 
 [dependencies]
-bevy = { version = "0.13.0", default-features = false }
+bevy = { version = "0.13.2", default-features = false }
```
Comment (stargazing-dino) [inline Cargo.toml:18]: I'm not sure if it's customary to bump these minor versions in libraries. Local setups usually handle choosing the right version. not sure though

**Review by stargazing-dino — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (ekalosak) [reply Cargo.toml:18]: Thx for comment. I will happily revert if indeed bad pattern.

Comment (stargazing-dino) [reply Cargo.toml:18]: I tested locally and it looks like it works with either 0.13.0 or 0.13.2. Again, not sure if bad pattern or not, just something I might've seen/heard elsewhere

**Comment by stargazing-dino** [comment]
I reproduced your initial issue and verified these changes fixed it for me

**Comment by zkat** [comment]
me, with this bug:
images

anyway, thanks for the PR.

**Comment by ekalosak** [comment]
Ah, so, I can verify that the change to the import path has _nothing_ to do with it.

The fix is to make sure that the Bevy version used by Big Brain match the Bevy version used by the user code. So, with latest Bevy 0.13 and latest Big Brain 0.19, there is a mismatch - as Big Brain 0.19 uses Bevy 0.12.

Thus the compiler saying "found a different Commands" - it's from a semantically different Bevy dependency.

In my user code, I simply specify the Big Brain dependency using a git location - as the main:HEAD of this project is on Bevy 0.13 - and the error evaporates, the example builds successfully, and everything works.


Really bottom line just points to releasing Big Brain 0.20 including Bevy 0.13, imho.

**Comment by ekalosak** [comment]
The compiler message could be more clear, I think is the real answer...

**Pull request merged**